RoslynPluginGenerator/CommandLine/ArgumentProcessor.cs

```diff
@@ -186,28 +185,17 @@ private NuGetReference TryParseNuGetReference(string argumentValue)
             return new NuGetReference(packageId, packageVersion);
         }
 
-        private bool TryParseSqaleFile(IEnumerable<ArgumentInstance> arguments, out string sqaleFilePath)
+        private bool TryParseSqaleArgument(IEnumerable<ArgumentInstance> arguments)
```

Comment:
Changed to write out an error and fail the argument processing of the user specifies the _/sqale_ parameter. There's a new unit test for this.

The error message tells the user what to do i.e. update the _rules.xml_, or use v1.0 of the SDK for old versions of SQ.